cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaSpecChecker.java

```diff
@@ -82,8 +84,8 @@ List<Condition> run() {
      * @param warnings List to add a warning to, if appropriate.
      */
     private void checkKafkaReplicationConfig(List<Condition> warnings) {
-        String defaultReplicationFactor = kafkaCluster.getConfiguration().getConfigOption(KafkaConfiguration.DEFAULT_REPLICATION_FACTOR);
-        String minInsyncReplicas = kafkaCluster.getConfiguration().getConfigOption(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG);
+        Object defaultReplicationFactor = kafkaSpec.getKafka().getConfig().get(KafkaConfiguration.DEFAULT_REPLICATION_FACTOR);
+        Object minInsyncReplicas = kafkaSpec.getKafka().getConfig().get(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG);
```

Comment:
Do we need a null check for `kafkaSpec.getKafka().getConfig()` or is it done some where before.